### ISSUE TITLE
Should we change the Model naming convention?

### ISSUE THREAD

**Issue by lucasmedeirosleite** [dailydrip/ecto-activerecord#2]
In `Ecto` earlier versions "deprecated" the `Model` naming in flavor of `Schema`, shouldn't we change it here too?

**Comment by knewter** [comment]
Yeah, that makes sense. Honestly, the word `Model` shouldn't even really appear since this is entirely about queries. Just needs some tweaking to the examples to make this change.

**Comment by franzejr** [comment]
@lucasmedeirosleite can you do this?
